app/utils/validators/forms/supplementary/ItemTypeValidator.scala

```diff
@@ -67,14 +68,13 @@ object ItemTypeValidator extends Validator[ItemType] {
   private val mappingNationalAdditionalCode = seq(
     text()
       .verifying(
-        "declaration.itemType.nationalAdditionalCode.error.length",
-        noLongerThan(nationalAdditionalCodeMaxLength)
+        "declaration.itemType.nationalAdditionalCode.error.invalid",
+        isContainedIn(NationalAdditionalCode.all.map(_.value))
       )
-      .verifying("declaration.itemType.nationalAdditionalCode.error.specialCharacters", isAlphanumeric)
   ).verifying(
-      "declaration.itemType.nationalAdditionalCode.error.maxAmount",
-      codes => codes.size <= nationalAdditionalCodesMaxAmount
-    )
+    "declaration.itemType.nationalAdditionalCode.error.maxAmount",
+    codes => codes.size <= nationalAdditionalCodesMaxAmount
```

Comment:
Not sure this is technically needed as there isnt even 99 codes to use but I've left it in incase the csv grows in the future